frame/support/src/dispatch.rs

```diff
@@ -2687,6 +2844,23 @@ mod tests {
 		assert_eq!(<Module<TraitImpl> as OnInitialize<u32>>::on_initialize(10), 7);
 	}
 
+	#[test]
+	#[should_panic(expected = "on_idle")]
+	fn on_idle_should_work_1() {
+		<Module<TraitImpl> as OnIdle<u32>>::on_idle(42, 9);
+	}
+
+	#[test]
+	#[should_panic(expected = "on_idle")]
+	fn on_idle_should_work_2() {
+		<Module<TraitImpl> as OnIdle<u32>>::on_idle(9, 42);
+	}
+
+	#[test]
+	fn on_idle_should_work_3() {
+		assert_eq!(<Module<TraitImpl> as OnIdle<u32>>::on_idle(10, 11), 7);
+	}
+
```

Comment:
what are these tests testing? why is 42 panic relevant?

Reply:
So the goal of these tests were to make sure they fire and they they properly pass the parameters in. I agree this could have been done differently but all the other tests were done with the panic and I was like i'll keep that standard